Dockerfile.assisted-test-infra

```diff
@@ -1,4 +1,15 @@
-FROM quay.io/edge-infrastructure/assisted-service:latest AS service
+# Generate python client
+FROM quay.io/edge-infrastructure/swagger-codegen-cli:2.4.18 AS swagger_py
+
+COPY assisted-service/swagger.yaml /
```

Comment:
Can we retrieve this file without relying on a checkout? Maybe we can pass branch name as build argument?

Reply:
what checkout do you mean ?

Reply:
`assisted-service` subfolder is being checked out by assisted-test-infra. We don't really need to rely on this for this dockerfile